plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -240,10 +242,22 @@ export class HookCommander {
                 text: messageMarkdown,
             }
         }
+        const statsd = this.statsd
+        ;(AbortSignal as any).timeout ??= function timeout(ms: number) {
```

Comment:
Monkey-patching `AbortSignal` globally makes me feel dirty… Let's just use `addEventListener`  for monitoring (https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/abort_event#examples)